lib/internal/validators.js

```diff
@@ -473,7 +473,7 @@ function validateLinkHeaderFormat(value, name) {
     throw new ERR_INVALID_ARG_VALUE(
       name,
       value,
-      'must be an array or string of format "</styles.css>; rel=preload; as=style"'
+      'must be an array or string of format \'</styles.css>; rel=preload; as="style"\''
```

Comment:
Can you undo this change? We try to avoid changing error messages when possible because users sometimes match on them. They shouldn't but they do.

That said, I personally don't see the point of validating the Link header beyond what `validateHeaderValue()` does. It's output from the program, it's trusted implicitly. Consider getting rid of the regex completely.